### ISSUE TITLE
Unrecognized regex (?^n:.). Please submit a bug report for Data::Printer

### ISSUE THREAD

**Issue by darviarush** [garu/Data-Printer#141]
In perlre is n-flag / [https://perldoc.perl.org/perlre.html#`*n*`](https://perldoc.perl.org/perlre.html#*n*)

```sh

$ perl -MCarp -MDDP -e '$SIG{__DIE__} = sub { Carp::confess @_ }; p my $x=qr/./n'
Unrecognized regex (?^n:.). Please submit a bug report for Data::Printer. at -e line 1.
 at -e line 1.
	main::__ANON__("Unrecognized regex (?^n:.). Please submit a bug report for Da"...) called at /usr/share/perl/5.26/Carp.pm line 167
	Carp::croak("Unrecognized regex (?^n:.). Please submit a bug report for Da"...) called at /usr/local/share/perl/5.26.1/Data/Printer.pm line 637
	Data::Printer::Regexp(qr(.)n, HASH(0x556e726957a0)) called at /usr/local/share/perl/5.26.1/Data/Printer.pm line 281
	Data::Printer::_p(qr(.)n, HASH(0x556e726957a0)) called at /usr/local/share/perl/5.26.1/Data/Printer.pm line 498
	Data::Printer::REF(REF(0x556e72695908), HASH(0x556e726957a0)) called at /usr/local/share/perl/5.26.1/Data/Printer.pm line 281
	Data::Printer::_p(REF(0x556e72695908), HASH(0x556e726957a0)) called at /usr/local/share/perl/5.26.1/Data/Printer.pm line 247
	Data::Printer::_data_printer("", REF(0x556e72695908)) called at /usr/local/share/perl/5.26.1/Data/Printer.pm line 152
	Data::Printer::p(REF(0x556e72695908)) called at -e line 1
```

**Comment by garu** [comment]
Hello! Thank you for using Data::Printer and for taking the time to file this report. I could have sworn it was fixed on 1.0.0 but I totally missed it. It's now fixed on the repo (commit 12d44aa) and I'll push 1.0.1 with the fix to CPAN as soon as I finish looking at all open issues so I can bundle any other fix together.

Thanks again!

**Comment by garu** [comment]
Done! Thanks again for reporting the issue <3
